tests/aws/services/events/conftest.py

```diff
@@ -194,23 +194,12 @@ def _delete_log_group():
 
 
 @pytest.fixture
-def put_events_with_filter_to_sqs(aws_client, sqs_get_queue_arn, clean_up):
+def create_sqs_events_target(aws_client, sqs_get_queue_arn):
     queue_urls = []
-    event_bus_names = []
-    rule_names = []
-    target_ids = []
-
-    def _put_events_with_filter_to_sqs(
-        pattern: dict,
-        entries_asserts: list[Tuple[list[dict], bool]],
-        input_path: str = None,
-        input_transformer: dict[dict, str] = None,
-    ):
-        queue_name = f"queue-{short_uid()}"
-        rule_name = f"rule-{short_uid()}"
-        target_id = f"target-{short_uid()}"
-        bus_name = f"bus-{short_uid()}"
 
+    def _create_sqs_events_target(queue_name: str | None = None) -> tuple[str, str]:
+        if not queue_name:
+            queue_name = f"tests-queue-{short_uid()}"
         sqs_client = aws_client.sqs
         queue_url = sqs_client.create_queue(QueueName=queue_name)["QueueUrl"]
```

Comment:
This on purpose does not use the `sqs_create_queue` fixture here, since it would need to be moved to helper functions because no cleanup and thus no factory would be required, which in turn would not valid this to be a fixture. This would in turn require passing in the aws_client as input.
I believe it is easier to read if the queue is created with the boot client here directly.